cassandra-diagnostics-connector30/src/integration-test/java/io/smartcat/cassandra/diagnostics/connector/ITConnector.java

```diff
@@ -22,27 +23,32 @@
 
     private static Cluster cluster;
     private static Session session;
-    private static CountDownLatch lock = new CountDownLatch(1);
-    private static boolean queryIntercepted;
+    private static CountDownLatch lock = new CountDownLatch(2);
+    private static int queryInterceptedCnt;
```

Comment:
I would rather spend two more letters and make this clear `queryInterceptedCount`